integration/spark/src/main/scala/org/apache/spark/sql/execution/command/carbonTableSchema.scala

```diff
@@ -390,11 +390,19 @@ case class LoadTable(
       val useKettle = options.get("use_kettle") match {
         case Some(value) => value.toBoolean
         case _ =>
-          val useKettleLocal = System.getProperty("use.kettle")
+          var useKettleLocal = System.getProperty("use.kettle")
+          if (useKettleLocal == null && sqlContext.sparkContext.getConf.contains("use_kettle")) {
```

Comment:
It is strange that "use.kettle", "use_kettle", and "useKettle" are all different, can we use the same name for all  these  configuration?